src/features/search/components/SearchHeaderTabs/SearchHeaderTabs.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from 'react';
+import React, { memo } from 'react';
+import { Tabs } from 'src/components/Tabs';
+
+const tabs = {
+  vn: 'Visual novels',
+  release: 'Releases',
+  character: 'Characters',
+};
+
+export type TabValue = keyof typeof tabs;
+
+const tabValues = Object.keys(tabs) as readonly TabValue[];
+
+const disabledTabs: readonly TabValue[] = ['release', 'character'];
+
+interface Props {
+
+  /** Active tab value. */
+  readonly value: TabValue;
+
+  /** Change callback. */
+  readonly onChange?: (value: TabValue) => void;
+}
+
+/** Tabs. */
+const SearchHeaderTabsComponent: FC<Props> = ({
+  value,
+  onChange,
+}) => (
+  <Tabs.Root
+    asChild
+    activationMode="manual"
+    value={value}
+
+    // Making tabs think that it works with `strings`, but it actually works with `TabValue` union.
```

Comment:
:)

Reply:
I think we should create wrapper for tabs, that encapsulate such logic. It probably should take readable map object as input.
Please add todo for this.

Reply:
Added todo